appveyor.yml

```diff
@@ -173,6 +173,7 @@ on_success:
 
 
 on_finish:
+  - appveyor PushArtifact junit-results.xml
```

Comment:
This not a  duplicate, but code that appveyor requested for debugging purposes. I will remove it.